src/message/MessageList.js

```diff
@@ -18,10 +19,25 @@ const styles = StyleSheet.create({
 export default class MessageList extends React.PureComponent {
   autoScrollToBottom = false;
 
+  constructor() {
+    super();
+    this.state = {
+      longPressedMessage: null,
```

Comment:
If you can do it old-school. For example, something happens, and in the handler you just do `doSomething()`. That would be much preferrable to just setting a state and then the component reacting on the state.